webpack.config.js

```diff
@@ -16,13 +16,17 @@ module.exports = env => {
           exclude: [
             path.resolve(__dirname, 'node_modules')
           ],
-          use: [
-            'babel-loader',
-          ]
+          use: {
+            loader: 'babel-loader',
+            options: {
+              presets: ['@babel/preset-env'],
```

Comment:
One concern is that things defined in our webpack.config don't propagate to downstream packages. We should minimize what's needed, because e.g. the secret ballet would also need to modify it's webpack setup to use these same plugins / presets / etc.

Reply:
`@babel/plugin-syntax-dynamic-import` is needed to support the `import()` syntax and `@babel/plugin-transform-regenerator` is needed to support async/await